util/Migrator/DbScripts/2020-08-12_00_OrgIdentifierProc.sql

```diff
@@ -5,7 +5,7 @@ END
 GO
 
 CREATE PROCEDURE [dbo].[Organization_ReadByIdentifier]
-    @Identifier UNIQUEIDENTIFIER
+    @Identifier NVARCHAR(50)
```

Comment:
We should correct this with a new script at this point.

Reply:
Done